updater/spago.dhall

```diff
@@ -0,0 +1,17 @@
+{ name = "contrib-updater"
+, dependencies =
+  [ "argonaut-core"
+  , "codec-argonaut"
+  , "console"
+  , "effect"
+  , "heterogeneous"
+  , "interpolate"
+  , "node-fs-aff"
+  , "node-process"
+  , "optparse"
+  , "psci-support"
+  , "sunde"
+  ]
+, packages = ./packages.dhall
+, sources = [ "src/**/*.purs", "test/**/*.purs" ]
```

Comment:
```suggestion
, sources = [ "src/**/*.purs" ]
```
Unless we're planning on writing tests.

Reply:
I'd like to keep this in as a tacit reminder to eventually write them.